pyServer/AzureDiskInspectService.py

```diff
@@ -85,6 +85,14 @@ def getContainerId():
 class ResponseHeaderMetadata:
     KEEPALIVETHREAD_TIMEOUT_IN_MINS = "KeepAliveThread-Timeout-In-Mins"
 
+"""
+Error response model
+"""
```

Comment:
Add some details like: "Error response entity model in compliance with ARM RPC protocol..." etc

Reply:
Updated comment as suggested